src/components/views/dialogs/CreateRoomDialog.tsx

```diff
@@ -95,7 +107,7 @@ export default class CreateRoomDialog extends React.Component<IProps, IState> {
             createOpts.creation_content = { 'm.federate': false };
         }
 
-        if (!this.state.isPublic) {
+        if (this.state.joinRule !== JoinRule.Public) {
```

Comment:
Should this be turned in an `else` statement instead?

Reply:
oh, yeah 🤦